index.bs

```diff
@@ -1,317 +1,307 @@
 
-	The .locked element's 'render-subtree' configuration
-	lets the user-agent manage rendering the subtree of the element.
-	Specifically when this element is near the viewport,
-	the user-agent will begin rendering the element.
-	When the element moves away from the viewport,
-	it will stop being rendered.
-
-	Recall that when not rendered,
-	the property also applies [=size containment=] to the element.
-	This means that when not rendered,
-	the element will use the specified 'contain-intrinsic-size',
-	making the element layout as if it had a single block child with 100px width and 200px height.
-	This ensures that the element still occupies space when not rendered.
-	At the same time,
-	it lets the element size to its true contents when the subtree is rendered
-	(since [=size containment=] no longer applies),
-	thus removing the concern that estimates like 100x200 are sometimes inaccurate
-	(which would otherwise result in displaying incorrect layout for on-screen content).
+  The .sv element's 'subtree-visibility' value [=auto=] lets the user-agent
+  manage whether the element is [=skipped=].  Specifically when this element is
+  near the viewport, the user-agent will begin painting the element.  When the
+  element moves away from the viewport, it will stop being painted. In
+  addition, the user-agent should skip as much of the rendering work as
+  possible when the element is [=skipped=].
 </div>
 
 <div class=example>
 	<xmp highlight=markup>
 		<style>
-		.locked {
-		  render-subtree: invisible skip-activation;
-		  contain-intrinsic-size: 100px 200px;
+		.sv {
+		  subtree-visibility: hidden;
 		}
 		</style>
 
-		<div class=locked>
+		<div class=sv>
 		  ... some content goes here ...
 		</div>
 	</xmp>
 
-	In this case, the rendering of the subtree is managed by the developer only.
-	This means that if script does not modify the value,
-	the element's subtree will remain unrendered,
-	and it will use the 'contain-intrinsic-size' input when deciding how to size the element.
+  In this case, the element is [=skipped=] regardless of viewport intersection.
+  This means that the only way to have this subtree painted is via script
+  updating the value to remove 'subtree-visibility' or change its value. As
+  before, the user-agent should skip as much of the rendering in the subtree as
+  possible.
+
+  An additional effect of skipping rendering is that the layout state of the
+  subtree can be preserved by the user-agent, so that removing the
+  'subtree-visibility' property in the future will cause the subtree to be
+  rendered quicker than otherwise possible.
 </div>
 
 <div class=example>
 	<xmp highlight=markup>
 		<style>
-		.locked {
-		  render-subtree: invisible skip-viewport-activation;
+		.sv {
+		  subtree-visibility: hidden;
 		}
 		</style>
 
-		<div class=locked>
+		<div id=target class=sv>
 		  ... some content goes here ...
 		</div>
+    <script>
+      ...
+      target.firstElementChild.getBoundingClientRect();
```

Comment:
Add a comment that this will force layout to occur if the UA had previously avoided it.

Reply:
Also, add enough CSS details to the example that you can add to the comment the exact value that getBoundingClientRect() will return.